src/openxr/OpenXRApi.cpp

```diff
@@ -910,29 +882,31 @@ bool OpenXRApi::initialiseSession() {
 		return false;
 	}
 
-#ifdef ANDROID
-	// TODO: Instead of switching on Android, use the new validation mechanism to determine when this should be configured.
+	// always define, we ignore this if it's not used.
 	XrSystemColorSpacePropertiesFB color_space_properties_fb = {
 		.type = XR_TYPE_SYSTEM_COLOR_SPACE_PROPERTIES_FB,
 	};
```

Comment:
Here and below, shouldn't the `XR` structs be defined under their corresponding `if` block?

Reply:
Uhm, no, because they go out of scope before they are used and that will crash the application. I've made that mistake a few times :P

It's a little ugly that as a result unused structs are defined but it is what it is.

Reply:
I wasn't aware, thanks for the tip!